### PR TITLE
Upgrade vue-cli-service

### PR DESCRIPTION
**Pull request by zachmullen**
I'm doing the yarn upgrade incrementally. This will help us isolate what package upgrades are causing the test failures, as well as begin to eliminate our security warnings for third party packages.

### CODE REVIEW

**Comment by subdavis** [comment]
@zachmullen do you want to rebase once #238 is merged, or is it able to merge now?

**Comment by zachmullen** [comment]
This is ready for review. After it's merged I'll work on a full `yarn upgrade` PR.

**Comment by zachmullen** [comment]
The tests are failing related to vuetify import, but are passing for me locally...

**Comment by zachmullen** [comment]
This is ready for review.

File: demo/App.vue [hunk 1]
```diff
@@ -348,7 +357,7 @@ export default {
     detailsList() {
       if (this.selected.length) {
         return this.selected;
-      } else if (this.location && this.location._id) {
+      } if (this.location && this.location._id) {
```
Comment (subdavis) [inline demo/App.vue]: wait huh?

Comment (zachmullen) [reply demo/App.vue]: Yes, this was auto-linted away due to the `return`.

**Pull request merged**